### ISSUE TITLE
Commands d, da, l, la, etc. are inconsistent.

### ISSUE THREAD

**Issue by karlic** [dhutchison/PythonistaScripts#24]
Having looked at the prompts and commands, it seems that d/da has three different meanings/functions and l/la two. I think this makes their use confusing and could lead to mistakes. Could they be renamed to make them more intuitive?

```
File %s is in the sync cache and on Dropbox, but no longer exists locally. (Default Delete):
Delete From Dropbox (d) [All in this state (da)]
Download File (l) [All in this state (la)]
Skip (s) [All in this state (sa)]

File %s has been updated both locally and on Dropbox. (Default Skip) Overwrite: 
Dropbox Copy (d) [All in this state (da)]
Local Copy (l) [All in this state (la)]
Skip (s) [All in this state (sa)]

File %s is not in the sync cache but exists both locally and on dropbox. (Default Skip) Overwrite:
Dropbox Copy (d) [All in this state (da)]
Local Copy (l) [All in this state (la)]
Skip (s) [All in this state (sa)]

File %s is in the sync cache but no longer on Dropbox. 
(Default Delete):
Delete local file (d) [All in this state (da)]
Upload File (u) [All in this state (ua)
Skip (s) [All in this state (sa)]

```

**Comment by dhutchison** [comment]
I'm open to suggestions for new prompts, got any ideas? (I'd never considered the prompts as a whole as they are different situations, and I generally only use it from 1 device so don't have many conflicts to resolve)

**Comment by karlic** [comment]
I've updated the code above to add the prompts that the responses refer to.

My thoughts are:
0. If you change the commands, do it so using the old command doesn't accidentally delete!
1. A default of delete is dangerous. I'm often syncing to test just one file and could (think I was) skip in haste and lose a file(s).
2. 1. is worse because sometimes skip (the absolute opposite to delete) is the default.
3. "Overwrite?" is counterintuitive (to me)---I'd expect to choose the file I want to keep NOT lose.
4. Consistently differentiating Dropbox and Local is good (so the first group should say Local somewhere).
4a. Should Local be called Pythonista?
5. Don't have 3 'd's Dropbox, Delete, Download. (E.g., Dropbox=Remote, Delete=Remove, Download=Create Local??)
6. Have Dropbox/Local in same position in command. (1st group has it a the end, "Delete from Dropbox"; 2nd has it at start "Dropbox Copy".
7. The Skip command is good in all groups.
8. It's not easy to get right. :-/ 
9. Are there any other sync programs' menus you can borrow from?

Here's one suggestion, which still doesn't address all the above.

```
File %s is in the sync cache and on Dropbox, but no longer exists locally. 
(Default is Skip):
Remove from Dropbox (r) [All in this state (ra)]
Copy to Local file (c) [All in this state (ca)]
Skip (s) [All in this state (sa)]

File %s has been updated both locally and on Dropbox.
(Default is Skip):
Dropbox replaces Local (d) [All in this state (da)]
Local replaces Dropbox (l) [All in this state (la)]
Skip (s) [All in this state (sa)]

File %s is not in the sync cache but exists both locally and on Dropbox. 
(Default is Skip):
Dropbox replaces Local (d) [All in this state (da)]
Local replaces Dropbox (l) [All in this state (la)]
Skip (s) [All in this state (sa)]

File %s is in the sync cache but no longer on Dropbox. 
(Default is Skip):
Remove Local file (r) [All in this state (ra)]
Copy to Dropbox  (c) [All in this state (ca)
Skip (s) [All in this state (sa)]

```

**Comment by dhutchison** [comment]
Sorry, I've only just got around to looking at this.

I agree with the points you made, but the new prompts do not read right.

**Comment by karlic** [comment]
I agree they're not quite right. I was trying to make the abbreviation match a verb, but 
`Dropbox replaces Local (d) [All in this state (da)]`
obviously doesn't.

Also, `(r)` and `(c)` are used to refer to Dropbox in one case and Local in another, which isn't ideal. 

tl;dr: I don't have a perfect answer, or would have submitted a pull request. ;-)
